app/assets/src/stylesheets/02-molecules/forms.scss

```diff
@@ -7,6 +36,23 @@ input[type=checkbox] + label {
   margin-bottom: 0;
 }
 
-.field, form > .actions {
+.field form > .actions {
```

Comment:
This change looks accidental. Explain what it's doing or remove it, please.

Reply:
Complete accident.